public/js/monitor.js

```diff
@@ -58,76 +51,15 @@ function doOauth(el, {emailWatch = false} = {}) {
     return;
   }
```

Comment:
Leaving this function – it quickly exits the function if session storage is unavailable.